ChromeDevToolsClient/src/main/java/com/hubspot/chrome/devtools/client/ChromeWebSocketClient.java

```diff
@@ -90,7 +90,9 @@ public void onMessage(String message) {
           executorService.submit(() -> eventListener.onEvent(type, event));
         }
       } else if (response.isError()) {
-        LOG.error(response.getError().toString());
+        String errorMessage = response.getError().toString();
+        LOG.error(errorMessage);
+        throw new ChromeDevToolsException(errorMessage, response.getError().getCode());
```

Comment:
just double checking, since it's inside the websocket client here, does this throw all the way back up to the line originally calling the method?

Reply:
good catch, no it didn't. fixed now